README.md

```diff
@@ -52,8 +52,10 @@ Once you have `allennlp` installed, run the following within the same Python env
 
 ```
 git clone https://github.com/allenai/allennlp-models.git
-EXCLUDE_ALLENNLP_IN_SETUP=true pip install -e .
+ALLENNLP_VERSION_OVERRIDE='allennlp' pip install -e .
```

Comment:
sorry this is an unrelated change, but you probably also want to add a `cd allennlp-models` above this line?